db/migrate/20241030160457_create_bulk_update_trainee_upload_rows.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class CreateBulkUpdateTraineeUploadRows < ActiveRecord::Migration[7.2]
+  def change
+    create_table :bulk_update_trainee_upload_rows do |t|
+      t.references :bulk_update_trainee_upload, null: false, foreign_key: true
+      t.integer :row_number, null: false
+      t.jsonb :data, null: false
+      t.jsonb :error_messages
```

Comment:
Is there a use case for the `error_messages` if we use the `BulkUpdate::RowError` instead?